registry/zookeeper/registry.go

```diff
@@ -381,7 +381,11 @@ func (r *zkRegistry) registerTempZookeeperNode(root string, node string) error {
 	}
 	zkPath, err = r.client.RegisterTemp(root, node)
 	if err != nil {
-		logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		if err == zk.ErrNodeExists {
+			logger.Warnf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		} else {
+			logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		}
 		return perrors.WithMessagef(err, "RegisterTempNode(root{%s}, node{%s})", root, node)
```

Comment:
Is condition (err==zk.ErrNodeExists )should return perrors.WithMessagef?

Reply:
> This function will return error. And outer function will catch the error ,continue to call logger.Error. If you need to ignore the error, I think you should modify all error catch.

For now, there is no way to print this error except by calling the Interface Methods peripherally.